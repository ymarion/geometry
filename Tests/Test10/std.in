# Initializing the objects
C c1 0 0 10
R r1 0 0 20 20
L l1 0 0 20 20
PL pl1 0 0 0 15 15 15 15 0
# And now, for something completely different, the enum !
LIST
#
EXIT
