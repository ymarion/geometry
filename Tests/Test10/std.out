# Creations
OK
OK
OK
OK
# Alphabetical order
C c1 0 0 10
L l1 0 0 20 20
PL pl1 0 0 0 15 15 15 15 0
R r1 0 0 20 20
